react-components/src/components/Architecture/CommandButton.tsx

```diff
@@ -23,6 +23,7 @@ export const CommandButton = ({
   const { t } = useTranslation();
   const command = useMemo<BaseCommand>(() => getDefaultCommand(inputCommand, renderTarget), []);
 
+  // @update-ui-component-pattern
```

Comment:
I think, we can move update-ui-component-pattern as a separate file as it is used in several components, your thoughts?

Reply:
I want, but do not know how in the react framework. Any suggestions? I have to use all the states in the rest of the code.

Reply:
Anyway, this is a fix. I rather do this on another PR. Understand?